src/components/wizard-navigation-bar.component.horizontal.less

```diff
@@ -83,32 +112,29 @@ aw-wizard-navigation-bar.horizontal {
           .line(@small-dot-width, @small-dot-height, @wz-color-default);
         }
 
-        &:after {
+        .step-indicator {
           .state-circle(@small-dot-width, @small-dot-height, 0);
           .state-circle-with-background(@wz-color-default);
         }
-      }
 
-      // default steps shouldn't change when hovered, because they aren't clickable
-      li.default a:hover {
-        color: @wz-color-current;
+        a:hover .step-indicator {
+          .state-circle-hover(@small-dot-width, @small-dot-height, 0);
+          .state-circle-with-background-hover(@wz-color-default);
+        }
       }
 
-      li.current:after {
-        .state-circle-with-background(@wz-color-current);
-      }
+      li.current .step-indicator { .state-circle-with-background(@wz-color-current); }
```

Comment:
When are you using the single-line style and when a multi-line style?
I mean, hear you use:
```less
li.current .step-indicator { .state-circle-with-background(@wz-color-current); }
```
while in other places
```less
.state-circle-with-background-nohover(@circle-color) {
    background-color: @circle-color;
}
```
is used.

Reply:
Here and in a couple of other places there is a number of similar CSS rules in a row. I think that collapsing each of them into a single line improves readability and allows to skim through the code quicker. But this is subjective and arguable, I am perfectly fine with changing this if you have a different opinion.

Reply:
I'm ok with either way.
I just think we need some rules, when to use the single line style and when to use the multi line style.

Reply:
Well, it seems hard for me to define a rule which would not be subjective. I suggest that I convert all definitions to multi-line to avoid having a debate about coding style :)

Reply:
Done: ebe399b